glib/src/subclass/signal.rs

```diff
@@ -471,12 +464,22 @@ impl<'a> SignalBuilder<'a> {
     /// This is called if multiple signal handlers are connected to the signal for accumulating the
     /// return values into a single value.
     pub fn accumulator<
-        F: Fn(&SignalInvocationHint, &mut Value, &Value) -> bool + Send + Sync + 'static,
+        R: ValueType,
+        F: Fn(&SignalInvocationHint, R, &R) -> ControlFlow<R, R> + Send + Sync + 'static,
```

Comment:
This now required you to take a copy of the accumulator if you just want to forward it to the return value

Reply:
Hmm, the only signature I could think that wouldn't create additional copies is something like:

```rust
A: ValueType,
V: FromValue<'_>,
F: Fn(&SignalInvocationHint, A, V) -> (Continue, Option<A>) + Send + Sync + 'static,

Reply:
Actually that's not so good, maybe this?
```rust
pub fn accumulator<
    T: FromValue<'_> + StaticType,
    R: ToValue + StaticType,
    F: Fn(&SignalInvocationHint, T, T) -> ControlFlow<Option<R>, Option<R>> + Send + Sync + 'static,
>(mut self, func: F) -> Self {
    assert_eq!(T::static_type(), R::static_type());
    // ...
}
```

Reply:
Changed it to the above, `None` can be returned to avoid the copy